agenda-webapps/src/main/webapp/vue-app/agenda-common/components/pending-invitation/AgendaPendingInvitationDrawer.vue

```diff
@@ -54,17 +55,38 @@ export default {
     open() {
       this.$refs.calendarPendingDatePolls.open();
       this.$nextTick().then(() => {
-        this.retrievePendingDatePolls();
+        this.retrieveDatePolls();
       });
     },
     loadMore(){
       this.limit += this.pageSize;
-      this.retrievePendingDatePolls();
+      this.retrieveDatePolls();
     },
-    retrievePendingDatePolls(){
+    retrieveDatePolls() {
+      if (eXo.env.portal.spaceId) {
+        const spaceId = eXo.env.portal.spaceId;
+        this.$spaceService.getSpaceById(spaceId, 'identity')
```

Comment:
it's better to retrieve space identity id only once in `created()` and not call this each time

Reply:
yep i told it when i call open method called in created